src/openloco/GameCommands.h

```diff
@@ -35,6 +47,45 @@ namespace openloco::game_commands
     uint32_t doCommand(int esi, const registers& registers);
     bool sub_431E6A(const company_id_t company, map::tile_element* const tile = nullptr);
 
+    inline void do_0(thing_id_t source, thing_id_t dest)
+    {
+        registers regs;
+        regs.bl = GameCommandFlag::apply;
+        regs.dx = source;
+        regs.di = dest;
+        doCommand(static_cast<int32_t>(GameCommand::vehicle_rearange), regs);
```

Comment:
There's a typo in the enum value used here:
```suggestion
        doCommand(static_cast<int32_t>(GameCommand::vehicle_rearrange), regs);
```